platform/node/src/node_geojson.hpp

```diff
@@ -1,13 +1,25 @@
 #include <mbgl/style/conversion/geojson.hpp>
 
+#include <string>
 namespace mbgl {
 namespace style {
 namespace conversion {
 
 template <>
-optional<GeoJSON> Converter<GeoJSON>::operator()(const v8::Local<v8::Value>&, Error& error) const {
-    error = { "not implemented" };
-    return {};
+optional<GeoJSON> Converter<GeoJSON>::operator()(const v8::Local<v8::Value>& value, Error& error) const {
+    v8::Local<v8::Value> local = value;
+
+    Nan::HandleScope scope;
+
+    v8::Local<v8::Object> JSON = Nan::To<v8::Object>(
```

Comment:
Use [`Nan::JSON.Stringify`](https://github.com/nodejs/nan/blob/master/doc/json.md#api_nan_json_stringify).

Reply:
Good call - we can use it [here](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/node/src/node_map.cpp#L154-L165) as well.